bin/dotd

```diff
@@ -3,25 +3,8 @@
 require 'cgi'
 require 'logger'
 require 'tmpdir'
-
-# Utility to strip consistent leading whitespace from heredoc strings, allowing
-# you to format your code more readably.
-#
-# Usage:
-#   <<-DOC.unindent
-#     Some text or other that I want to start in column 1.
-#         An actually indented line.
-#     This line still unindented.
-#   DOC
-#
-# from http://stackoverflow.com/a/5638187/5000129
-class String
-  # Strip leading whitespace from each line that is the same as the
-  # amount of whitespace on the least-indented line of the string.
-  def unindent
-    gsub /^#{scan(/^[ \t]+/).min_by(&:length)}/, ''
-  end
-end
+require_relative '../lib/cdo/cdo_cli'
+include CdoCli
```

Comment:
My understanding was that there were two motivations for that decision.

1. To not depend on loading our whole environment, which is slow.

    This change doesn't violate that principle, since CdoCli is itself very slim.  I'll add a comment in that file suggesting we keep it that way though!

2. To allow one to copy the `dotd` script anywhere and still run it.

   I used to do this and have found that I don't anymore (it doesn't really matter once it's running).  I will take a quick Slack poll and see if anyone else is doing this.